apps/docs/src/app/platform/component-docs/platform-combobox/platform-combobox-examples/address.ts

```diff
@@ -0,0 +1,47 @@
+import { Entity } from '@fundamental-ngx/platform';
+
+
+export class Address implements Entity {
+
+
+  constructor(public readonly uniqueName: string,
+              public readonly name: string,
+              public readonly lines: string,
+              public readonly city: string,
+              public readonly state: string,
+              public readonly postalCode: string,
+              public readonly phone: string,
+              public readonly fax: string,
+              public readonly email: string,
+              public readonly url: string,
+              public readonly country: string) {
+  }
+
+
+  getTypes(): any {
+    return {
+      uniqueName: String,
+      name: String,
+      lines: String,
+      city: String,
+      state: String,
+      postalCode: String,
+      phone: String,
+      fax: String,
+      email: String,
+      country: String
+    };
+  }
```

Comment:
You can create a type and return a type instead of any

Reply:
this method is from the parent file we are overriding the method, we cannot change the signature